src/main/java/io/cdap/plugin/cdc/source/salesforce/records/ChangeEventRecord.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.cdc.source.salesforce.records;
+
+import io.cdap.plugin.cdc.common.OperationType;
+import org.json.JSONObject;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Contains information about change event
+ */
+public class ChangeEventRecord {
```

Comment:
There is no salesforce class for this objects returned by their API? 

If there isn't, we should use Gson in combination with a Java class that directly represents the JSON response for the API, instead of doing custom parsing.